tests/integration/conftest.py

```diff
@@ -514,12 +516,18 @@ def _check_top_up_motion_enactment(
             top_up_recipients,
         )
 
-        assert sender_balance == sender_balance_before - spending
+        if top_up_token == lido_contracts.steth:
```

Comment:
can check also `sharesOf` avoiding tolerance issues (in addition, not a replacement)

Reply:
done